src/NuGet.Config

```diff
@@ -6,6 +6,9 @@
   <packageSources>
     <add key="myget.org dotnet-buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
+    <add key="myget.org dotnet-core" value="https://www.myget.org/F/dotnet-core/" />
+    <add key="myget.org dotnet-coreclr" value="https://www.myget.org/F/dotnet-coreclr/" />
```

Comment:
right this is my bad, I believe the only one we really need is the dotnet-core one to be able to restore the required packages for tool-runtime